src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator {
+    private String color;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String info() {
```

Comment:
```suggestion
    public String draw() {
```

and move to the separate interface, like AreaCalculator